server/src/permissions/index.ts

```diff
@@ -65,7 +65,7 @@ export const permissions = shield(
       structureDefinition: rules.isAuthenticatedUser,
     },
     Mutation: {
-      createAccessControl: rules.isAdmin,
+      createAccessControl: rules.isSourceWriter,
```

Comment:
Don't you need to add something so that the rule understands the userEmail parameter?

Reply:
well, the sourceId is still passed in this request and the userId is parsed from the JWT so no need to update this func :)

Reply:
Oh, right